tlstest/acl/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       source: ../mtls_internal/tls/alpha1
       target: /dgraph-tls
       read_only: true
-    command: /gobin/dgraph alpha --my=alpha1:7080 --zero=zero1:5080 --logtostderr -v=2
+    command: dgraph alpha --my=alpha1:7080 --zero=zero1:5080 --logtostderr -v=2
```

Comment:
Can the above /gobin bind point be removed?

Reply:
thanks - also forgot here